pkg/email/invite_user_template.html

```diff
@@ -0,0 +1,5 @@
+Hello,<br>
+You have been invited to join lakeFS<br>
+Click <a href="{{.URL}}">Here</a> to set up your account<br>
```

Comment:
In the previous version the address was visible - I think it is better to have the url you click in an email.
Suggest to check emails from online services for the email content.

Reply:
Making the URL visible makes it look very long and unprofessional,  Kept it as a link